### DISCUSSION TITLE
Hey Patrick it's EngrPips On Discord

### DISCUSSION THREAD

**Discussion by EngrPips** [Cyfrin/foundry-full-course-cu#1503] [General]
Hey @PatrickAlphaC , writing this to confirm my discord Identity

**Comment by mxnwafor** [comment]
hey man, thanks for your help today on discord. it's 0xavier.

**Reply by EngrPips** [reply]
Anytime buddy, I am glad i was able to help

**Comment by king-null** [comment]
hey @EngrPips please how can i reach you on discord

**Reply by EngrPips** [reply]
my discord is `engr.pips`, you can maybe send me a friend request.
